integration-tests/src/test/java/com/datastax/dse/driver/api/core/graph/GraphTestSupport.java

```diff
@@ -16,32 +16,35 @@
 package com.datastax.dse.driver.api.core.graph;
 
 import com.datastax.dse.driver.internal.core.graph.GraphProtocol;
+import com.datastax.oss.driver.api.testinfra.CassandraResourceRule;
+import com.datastax.oss.driver.api.testinfra.ccm.CcmRule;
 import com.datastax.oss.driver.api.testinfra.ccm.CustomCcmRule;
 import com.datastax.oss.driver.api.testinfra.session.CqlSessionRuleBuilder;
 
 /** Utility for creating commonly used Rule builders for tests. */
 public class GraphTestSupport {
 
-  /** CCM Rule builder for Graph Data Type tests. */
+  /**
+   * CCM Rule builder for serial and isolated Graph Data Type tests.
+   *
+   * <p>Only use this when you need additional configuration of the CCM rule. Otherwise, use the
+   * shared {@link CcmRule} (it has graph capability) and make the test parallelizable.
+   */
   public static final CustomCcmRule.Builder CCM_BUILDER_WITH_GRAPH =
       CustomCcmRule.builder()
           .withDseWorkloads("graph")
           .withDseConfiguration("graph.max_query_params", 32)
           .withDseConfiguration(
               "graph.gremlin_server.scriptEngines.gremlin-groovy.config.sandbox_enabled", "false");
 
-  /** CCM Rule builder for general Graph workload tests. */
-  public static final CustomCcmRule.Builder GRAPH_CCM_RULE_BUILDER =
-      CustomCcmRule.builder().withDseWorkloads("graph");
-
```

Comment:
I don't understand why this separate constant existed, I tried running all tests with the other one and it didn't seem to change anything. So unless there's some performance impact, I assume it's safe to run all tests with `sandbox_enabled`. That's what I did in `DefaultCcmBridgeBuilderCustomizer`.